### PR TITLE
Properly arrange channels for element-wise multi-pass

### PR DESCRIPTION
**Pull request by rotx-maxim**

### CODE REVIEW

File: izer/checkpoint.py [hunk 1]
```diff
@@ -33,12 +32,9 @@ def load(
 ):
     """
     Load weights and biases from `checkpoint_file`. If `arch` is not None and does not match
-    the architecuture in the checkpoint file, abort with an error message. If `fc_layer` is
-    `True`, configure a single fully connected classification layer for software rather than
-    hardware.
+    the architecuture in the checkpoint file, abort with an error message.
```
Comment (MaximGorkem) [inline izer/checkpoint.py:35]: typo: architecture

File: izer/checkpoint.py [hunk 2]
```diff
@@ -90,7 +83,7 @@ def load(
         if parameter in ['weight']:
             module, op = k.split(sep='.', maxsplit=1)
             op = op.rsplit(sep='.', maxsplit=1)[0]
-            if module != 'fc' or module == 'fc' and not fc_layer:
+            if module != 'fc' or module == 'fc':
```
Comment (MaximGorkem) [inline izer/checkpoint.py:86]: Could not understand this condition.

File: izer/onnxcp.py [hunk 3]
```diff
@@ -93,13 +92,10 @@ def load(
 ):
     """
     Load weights and biases from `checkpoint_file`. If `arch` is not None and does not match
-    the architecuture in the checkpoint file, abort with an error message. If `fc_layer` is
-    `True`, configure a single fully connected classification layer for software rather than
-    hardware.
-    `quantization` is a list of expected bit widths for the layer weights (always 8 for AI84).
+    the architecuture in the checkpoint file, abort with an error message.
```
Comment (MaximGorkem) [inline izer/onnxcp.py:95]: typo: architecture

**Review by MaximGorkem — CHANGES_REQUESTED** [review]
Looking good in overall, just small typos & an unnecessary condition

**Review by ermanok — APPROVED** [review]
Seems OK. I run the multipass add tests for 128 channels; they are running.

**Pull request merged**